src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/DoNotInitializeUnnecessarilyTests.cs

```diff
@@ -311,6 +312,65 @@ public class Test
                     .WithLocation(3));
         }
 
+        [Fact, WorkItem(5750, "https://github.com/dotnet/roslyn-analyzers/issues/5750")]
+        public async Task CA1805_ParameterlessValueTypeCtorAsync()
```

Comment:
Nit: the naming of this doesn't match the pattern used by the rest of the tests.